apps/www/components/Frame/Popover/Nav.js

```diff
@@ -58,7 +60,7 @@ const Nav = ({
                 </Button>
               </Link>
             )}
-            {me && (
+            {(hasActiveMembership || hasAccessGrants) && (
```

Comment:
Would use `hasAccess` instead, similar to onboarding hint check on account page.

`hasAccess` is provided by `withMe`. It checks for role `member`, which indicates whether you have access due to a membership or an access grant.

Reply:
that's so weird. because hasAccessGrant is used in various other places. Probably a case that could be cleaned a bit.

Reply:
What confused me: 
hasAccess
isMember
Are equivalent.
Some places use isMember, some isAccess. very confusing

Reply:
It is confusing.

Props avail largely depend on what `with…` a component is wrapped in e.g. `isMember` is provided by `withMembership` – albeit pretty much checking the same: has a user a `member` role.